src/components/kubectl/dashboard.ts

```diff
@@ -0,0 +1,180 @@
+'use strict';
+
+import * as vscode from 'vscode';
+import * as opn from 'opn';
+
+import { createReadStream } from 'fs';
+import { resolve } from 'path';
+import { fs } from '../../fs';
+import { portForwardToPod, PortMapping, buildPortMapping } from './port-forward';
+import { host } from '../../host';
+import { shell } from '../../shell';
+import { create as kubectlCreate, Kubectl } from '../../kubectl';
+
+
+const KUBE_DASHBOARD_URL = "http://localhost:8001/ui/";
+const TERMINAL_NAME = "Kubernetes Dashboard";
+const PROXY_OUTPUT_FILE = resolve(__dirname, 'proxy.out');
+
+const kubectl = kubectlCreate(host, fs, shell);
+const MAX_PORT_COUNT = 65535;
+
+// The instance of the terminal running Kubectl Dashboard
+let terminal:vscode.Terminal;
+
+/**
+ * Determines if the selected cluster is AKS or not by examining
+ * all the attached nodes for two heuristics:
+ * 1. Is every node an agent?
+ * 2. Is the node name prefixed with `aks-`? (TODO: identify if there's a better method for this/convince AKS team to add a label) for this.
+ * @returns Boolean identifying if we think this is an AKS cluster.
+ */
+async function isAKSCluster (): Promise<Boolean> {
```

Comment:
See https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html 

It's better to use `boolean` instead of `Boolean`.